articles/variadic-function-templates.dd

```diff
@@ -22,13 +24,12 @@ $(CONSOLE
 )
 
         $(P We'll explore how this can
-        be done in standard C++, followed by doing it using
-        the proposed variadic template C++ extension.
+        be done in C++.
         Then, we'll do it the various ways the D programming
         language makes possible.
         )
 
-$(H3 C++ Solutions)
+$(H3 $(LNAME2 cpp-solutions, C++ Solutions))
```

Comment:
This needs to be H2  (and consequently all H2 need to be H3) as the HEADERTOC generation will fail otherwise